### PR TITLE
[kinetic] use industrial_ci for travis

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Review by floweisshardt — APPROVED** [review]

**Review by floweisshardt — APPROVED** [review]

**Comment by floweisshardt** [comment]
@ipa-bnm: hat was the issue with having install tests?

**Comment by benmaidel** [comment]
@ipa-fmw I've disabled the test install because the .test file located at cob_script_server is failing. Please check and fix the `.test`. Afterwards we can reenable the test install.
It also looked like the test is never executed on the old travis scripts. The test should never pass because nothing is uploaded to the param server!

**Comment by floweisshardt** [comment]
ok, I'll check

File: .travis.rosinstall [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+    version: kinetic_dev
 - git:
     uri: 'https://github.com/ipa320/cob_extern.git'
     local-name: cob_extern
-    version: indigo_dev
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ros-industrial/ros_canopen.git'
+    local-name: ros_canopen
+    version: jade-devel
```
Comment (NadiaHG) [inline .travis.rosinstall:16]: jade-devel??

**Review by NadiaHG — DISMISSED** [review]

File: .travis.rosinstall (hunk 1 above)
Comment (benmaidel) [reply .travis.rosinstall:16]: yes, this is the latest version from ros_canopen. It should compile under kinetic

Comment (benmaidel) [reply .travis.rosinstall:16]: @ipa-nhg can you approve or comment, please

**Comment by benmaidel** [comment]
@ipa-nhg ping

File: .travis.rosinstall (hunk 1 above)
Comment (fmessmer) [reply .travis.rosinstall:16]: I approve, but alphabetic order in the .rosinstall file (cob_perception_common) would be nice :wink:

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +15,7 @@ env:
     - ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
 install:
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
-script:
-  - source .ci_config/travis.sh
+script: 
+  - .ci_config/travis.sh
```
Comment (fmessmer) [inline .travis.yml]: I like this notation here better, than the one in `indigo_dev` (see https://github.com/ipa320/cob_command_tools/pull/179/files#diff-354f30a63fb0907d4ad57269548329e3R17)!
(except for the trailing whitespace :wink:)

**Review by mgruhler — APPROVED** [review]

**Comment by fmessmer** [comment]
I will merge all [kinetic]-tagged PRs before cleaning up branches...

**Pull request merged**